src/gfx/gfxutils.js

```diff
@@ -436,6 +436,16 @@ function processMaterialForShadow(root, material) {
   });
 }
 
+function processGeoForSetRenderOrder(root, idMaterial) {
```

Comment:
I want to suggest simplification for the function name. What do you think about setGeoRenderOrder?

Reply:
Rename function to processObjRenderOrder